site/gatsby-site/src/pages/entities/edit.js

```diff
@@ -49,12 +49,14 @@ function EditEntityPage(props) {
     try {
       await updateEntityMutation({
         variables: {
-          query: {
-            entity_id: entityId,
+          filter: {
+            entity_id: { EQ: entityId },
           },
-          set: {
-            name: values.name,
-            date_modified: new Date(),
+          update: {
+            set: {
+              name: values.name,
```

Comment:
If entities had a `date_modified` field, please add it to the graphql schema :

`server/types/entity.ts`